sky/cli.py

```diff
@@ -3065,6 +3065,7 @@ def _down_or_stop_clusters(
             record['name']
             for record in all_clusters
             if controller_utils.Controllers.from_name(record['name']) is None
+            and (down or record['status'] != status_lib.ClusterStatus.STOPPED)
```

Comment:
```suggestion
             # do not select already stopped clusters if the action is to stop clusters.
             # stopped clusters are still included for down or autostop commands.
            and (down or idle_minutes_to_autostop is not None or record['status'] != status_lib.ClusterStatus.STOPPED)
```
i.e. I think we should still let (cancel) autostop/autodown operation to also act on stopped clusters - since those settings could apply when the cluster is brought back up again. I think this condition will do the job.
cc @cg505

Reply:
done!